k8s/aws/k8s-installer/vars.yml

```diff
@@ -5,7 +5,7 @@ cidr_block: 10.0.0.0/16
 cidr_block2: 10.0.1.0/24
 destination_cidr: 0.0.0.0/0
 image: ami-6b3fd60c
-master_size: t2.large
-node_size: t2.large
+master_size: t2.xlarge
```

Comment:
Master instance type may still be lesser than the nodes, I guess!

Reply:
okay! i'll assign the master to `t2.large`